reporter/src/funTest/kotlin/TestData.kt

```diff
@@ -112,6 +112,14 @@ val ORT_RESULT = OrtResult(
                                     id = Identifier("NPM:@ort:declared-license:1.0")
                                 )
                             )
+                        ),
+                        Scope(
+                            name = "devDependencies",
+                            dependencies = sortedSetOf(
+                                PackageReference(
+                                    id = Identifier("NPM:@ort:builder:1.0")
```

Comment:
You're referring to a package id that does not exist. You must use an existing id, like e.g. `"NPM:@ort:declared-license:1.0"`.